plugins/woocommerce/tests/e2e-pw/tests/merchant/order-status-filter.spec.js

```diff
@@ -60,7 +60,10 @@ test.describe( 'WooCommerce Orders > Filter Order by Status', () => {
 		// because tests are running in parallel, we can't know how many orders there
 		// are beyond the ones we created here.
 		for ( let i = 0; i < orderStatus.length; i++ ) {
-			const statusTag = 'text=' + orderStatus[ i ][ 0 ];
+			const statusTag = `.order-status.${ orderStatus[ i ][ 1 ].replace(
+				'wc-',
```

Comment:
Out of curiosity, in which scenarios have you found that the order status locator starts with `wc-` instead of `status-` @rodelgc? I've been testing it in both HPOS on and off, but always the locator with the `status-` prefix.

Reply:
The `wc-` locators were used to select the status filters for orders like in the screenshot below:
<img width="202" alt="rRYILAi9ol" src="https://user-images.githubusercontent.com/4509348/218978987-5a022cf0-bb72-4e23-bfe6-12fa19df4052.png">

This particular change was due to https://github.com/woocommerce/woocommerce/pull/36294 changing the spelling of "Cancelled" to "Canceled".

Reply:
I reverted this change because https://github.com/woocommerce/woocommerce/pull/36294 was reverted as well.